### PR TITLE
script/ceph-release-notes: work with py3 and remove backport release names from PRs

### PR DESCRIPTION
**Pull request by jdurgin**

### CODE REVIEW

File: src/script/ceph-release-notes [hunk 1]
```diff
@@ -51,6 +51,7 @@ prefixes = ['bluestore', 'build/ops', 'cephfs', 'cephx', 'cli', 'cmake',
 signed_off_re = re.compile("Signed-off-by: (.+) <")
 tracker_re = re.compile("http://tracker.ceph.com/issues/(\d+)")
 rst_link_re = re.compile(r"([a-zA-Z0-9])_(\W)")
+release_re = re.compile(r"^(nautilus|octopus|pacific):\s*")
```
Comment (djgalloway) [inline src/script/ceph-release-notes:54]: May as well add q?  Is that codename a secret?

Comment (jdurgin) [reply src/script/ceph-release-notes:54]: sure, doesn't hurt

**Review by djgalloway — APPROVED** [review]

**Comment by jdurgin** [comment]
jenkins test make check

**Review by batrick — APPROVED** [review]

**Pull request merged**